packages/shared/src/components/modals/common.tsx

```diff
@@ -167,6 +167,7 @@ export const modals = {
   [LazyModal.SquadPromotion]: SquadPromotionModal,
   [LazyModal.CreateSharedPost]: CreateSharedPostModal,
   [LazyModal.ReportPost]: ReportPostModal,
+  [LazyModal.ReportSquad]: <></>,
```

Comment:
Modal doesn't exist yet? (If so add a todo with ticket number please)

Reply:
Let me open a draft PR to make it easy to update the intermediate branch.

Reply:
Ok removing ✅